scripts/windows/buildBranch.bat

```diff
@@ -1,2 +1,2 @@
 @echo off
-powershell -command ".\build\Publish-Branch.ps1"
\ No newline at end of file
+powershell -command "%0\..\build\Publish-Branch.ps1"
```

Comment:
Would it make more sense to use `%~dp0\build\Publish-Branch.ps1` ?

Reply:
both should be good, but I changed. :)